packages/portal/backend/src/controllers/PersonController.ts

```diff
@@ -59,6 +62,51 @@ export class PersonController {
         return successful;
     }
 
+    public async processClasslist(personId: string = null, path: string = null,  data: any): Promise<Person[]> {
```

Comment:
I really really thing these classlist things shouldn't be in `PersonController` but in the `ClasslistAgent` class above. This would mean any change to the classlist service would change this class, but the abstraction of the agent should insulate this controller from any external service changes.

Reply:
We went with ClasslistAgent.